_data/sites.json

```diff
@@ -9283,6 +9283,17 @@
         ]
     },
 
+    {
+        "name": "One Month",
+        "url": "https://www.iubenda.com/privacy-policy/735465",
+        "difficulty": "hard",
+        "email": "support@honeybadger.io",
+        "notes": "Email support@honeybadger.io asking to close your account.",
```

Comment:
Is the e-mail right? Because it doesn't match the domain of one month, so it seems weird.

Reply:
Yes, as listed under their [privacy policy](https://www.iubenda.com/privacy-policy/735465):

![image](https://user-images.githubusercontent.com/76220359/190041073-107cc640-40fa-4ba6-b235-54695160d320.png)